openapi/api_v1.yml

```diff
@@ -78,14 +83,40 @@ paths:
                 500:
                     $ref: '#/components/responses/InternalServerErrorResponse'
 
+    /v1/auth/token/refresh:
```

Comment:
Nothing needed to be informed.
And the application knows only one thing - if the refresh token is valid, it can give you new access token.
That's all